### PR TITLE
Task 16

### PR DESCRIPTION
**Pull request by rudensergey**
Completed all task @baitun

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -107,6 +107,10 @@ <h6>You choose your own prices and will be in control of what your profit margin
         </div>
     </div>
 
+    <div>
```
Comment (baitun) [inline index.html]: What's the purpose of this empty div?

**Comment by baitun** [comment]
There is still `.DS_store` files

File: style/index.css [hunk 2]
```diff
@@ -36,4 +36,14 @@ li,
     font-weight: 300;
     margin-top: 0px;
     margin-bottom: 0px;
+}
+
+
+::-webkit-scrollbar {
```
Comment (baitun) [inline style/index.css]: Why do you use vendor prefix here?

File: index.html (hunk 1 above)
Comment (rudensergey) [reply index.html]: It was a test

File: style/index.css (hunk 2 above)
Comment (rudensergey) [reply style/index.css]: Ive seen this prefix on Stack Overflow as the best way to hide scroll bar

Comment (baitun) [reply style/index.css]: And why do you hide scrollbar?

Comment (rudensergey) [reply style/index.css]: It looks more beautiful🤘without scroll bars

Comment (baitun) [reply style/index.css]: How then should the user understand that there is more content below and he need to scroll?

Comment (rudensergey) [reply style/index.css]: page doesen't have 100vh image on the first screen which can stuck first user. Also when you see part of the elements below you always scroll down

Comment (baitun) [reply style/index.css]: It's not intuitive

Comment (rudensergey) [reply style/index.css]: first action which you do in the webpage? You don't see scroll bar but it doesen't matter you always scroll down

Comment (baitun) [reply style/index.css]: How shold I know how long this page is and where am I?
How long should I scroll it?
How do I know where is the end?

**Pull request merged**